internal/store/store.go

```diff
@@ -705,14 +695,17 @@ func (s *Store) PushComposeRequest(request ComposeRequest) error {
 		newJobs = append(newJobs, Job{
 			ComposeID:    request.ComposeID,
 			ImageBuildID: imageBuildID,
-			Distro:       distroString,
+			Distro:       request.Distro.Name(),
 			Manifest:     manifestStruct,
 			Targets:      []*target.Target{},
 			ImageType:    imgTypeCompatStr,
 		})
 
+		// this ought to exist, because we're creating it from an existing distro struct
+		distroTag, _ := common.DistributionFromString(request.Distro.Name())
```

Comment:
I'm not big fan of "ought to". I know it shouldn't happen, but I prefer error handling everywhere.

Reply:
But at this point, we're 100% certain that the tag exist. I don't think adding some dead code (`if ok`) is of any use.

(This code was removed in a later commit.)

Reply:
> (This code was removed in a later commit.)

 👍